ZeekPluginStatic.cmake

```diff
@@ -20,6 +20,16 @@ function(zeek_add_static_plugin ns name)
     endif()
     add_dependencies(${target_name} zeek_autogen_files)
 
+    # Skip zeek-version.h when including zeek-config.h for statically build
+    # plugins (they are always builtin) *except* if the current scope is tagged
+    # with ZEEK_BUILDING_EXTRA_PLUGINS (this is the case when building plugins
+    # from BUILTIN_PLUGIN_LIST).
+    if ( NOT ZEEK_BUILDING_EXTRA_PLUGINS )
```

Comment:
extra is now a 3rd variation on include and builtin. Those are the ones further up:
* ZEEK_INCLUDE_PLUGINS
* BUILTIN_PLUGIN_LIST

Maybe keep the builtin part and prolong `ZEEK_BUILDING_EXTRA_BUILTIN_PLUGINS`? I'd call them "builtin external plugins", but not sure this is terminology used elsewhere.